src/algorithms/crypto/randomness.id

```diff
@@ -1,8 +0,0 @@
-type DomainSeparationTag enum {
```

Comment:
Why did this disappear?

Reply:
Cleanly defined in `randomness.go` so again a question of legibility (minimizing imports to the md)